src/Columns/ColumnString.h

```diff
@@ -21,6 +21,11 @@ class SipHash;
 namespace DB
 {
 
+namespace ErrorCodes
+{
+    extern const int LOGICAL_ERROR;
+}
+
```

Comment:
bad - better move implementation to .cpp

Reply:
Yeah, good point. Moved the implementation to .cpp